paper/paper.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'scikit-optimize: Sequential model-based optimization toolkit.'
+tags:
+  - Python
+  - Machine-learning
+  - Global optimization
+  - Scientific Computing
+authors:
+ - name: Manoj Kumar
+   orcid: 0000-0002-3802-8346
+   affiliation: 1
+ - name: Tim Head
+   orcid:
+   affiliation: 2
+ - name: Gilles Louppe
+   orcid:
+   affiliation: 3
+ - name: Katie Malone
+   orcid:
+   affiliation: 4
+ - name: nel215
+   orcid:
+   affliation: 5
+ - name: Nuno Campos
+   orcid:
+   affliation: 6
+ - name: Zé Vinícius
+   orcid:
+   affliation: 7
+ - name: Thomas Fan
+   orcid:
+   affliation: 8
+ - name: Mikhail Pak
+   orcid:
+   affliation: 9
+ - name: Iaroslav Shcherbatyi
+   orcid:
+   affliation: 10
+ - name: Lilian Besson
+   orcid:
+   affliation: 11
+ - name: Fabian Linzberger
+   orcid:
+   affliation: 12
+ - name: Taylor Smith
+   orcid:
+   affliation: 13
+ - name: andreh7
+   orcid:
+   affliation: 14
+ - name: Mehdi Cherti
+   orcid:
+   affliation: 15
+ - name: Alexander Fabisch
+   orcid:
+   affliation: 16
+
+affiliations:
+ - name: New York University
+   index: 1
+ - name:
+   index: 2
+ - name:
+   index: 3
+ - name:
+   index: 4
+ - name:
+   index: 5
+ - name:
+   index: 6
+ - name:
+   index: 7
+ - name:
+   index: 8
+ - name:
+   index: 9
+ - name:
+   index: 10
+ - name:
+   index: 11
```

Comment:
Affiliation `11` for @Naereen: [CentraleSupélec](http://orcid.org/0000-0002-6972-589X)
(if the accent is messing up, you can use `{\'e}` if the `paper.md` file aims at being compiled to PDF through pandoc and LaTeX)